inbox/events/google.py

```diff
@@ -92,7 +93,7 @@ def sync_events(
         updates = []
         items = self._get_raw_events(calendar_uid, sync_from_time)
```

Comment:
Yes, this is true. The original Google implementaiton "hoards" the events into a single list, while it could actually  perfectly yield them one by one. Microsoft one does not have this problem as it uses generators all the way down the stack. I will port Google implementation to generators as well on a separate PR.

Reply:
I will change the name as well here.